packages/bundlers/experimental/src/ExperimentalBundler.js

```diff
@@ -251,11 +265,10 @@ function createIdealGraph(
     Array<[Dependency, Bundle]>,
   > = new DefaultMap(() => []);
 
-  // bundleRoot to all bundleRoot descendants
-  let reachableBundles: DefaultMap<
-    BundleRoot,
-    Set<BundleRoot>,
-  > = new DefaultMap(() => new Set());
+  // Connects bundleRoot assets to the assets that must be in the bundle
+  let reachableBundles: DefaultMap<BundleRoot, Set<Asset>> = new DefaultMap(
```

Comment:
`reachableBundles` doesn't seem to be used after getting populated.. should we move it?